hazelcast/test/src/compact_test.cpp

```diff
@@ -677,14 +843,19 @@ TEST_F(CompactNullablePrimitiveInteroperabilityTest,
     ASSERT_EQ(expected.nullableLong.value(), actual.long_);
     ASSERT_EQ(expected.nullableFloat.value(), actual.float_);
     ASSERT_EQ(expected.nullableDouble.value(), actual.double_);
+    ASSERT_SAME_CONTENT(expected.nullableBooleans, actual.booleans);
+    ASSERT_SAME_CONTENT(expected.nullableBytes, actual.bytes);
+    ASSERT_SAME_CONTENT(expected.nullableShorts, actual.shorts);
+    ASSERT_SAME_CONTENT(expected.nullableInts, actual.ints);
+    ASSERT_SAME_CONTENT(expected.nullableLongs, actual.longs);
+    ASSERT_SAME_CONTENT(expected.nullableFloats, actual.floats);
+    ASSERT_SAME_CONTENT(expected.nullableDoubles, actual.doubles);
 }
 
 TEST_F(CompactNullablePrimitiveInteroperabilityTest,
        testWriteNullReadPrimitiveThrowsException)
```

Comment:
In this test we are not testing try to reading every field throws, we are just asserting trying to read all fields throws or not, can we do anything about it 🤔 ?

Reply:
I thought the same thing as well. But I could not found a smart way of doing this without copy pasting test for each field type.